public/app/features/dashboard/dashnav/dashnav.ts

```diff
@@ -156,6 +159,22 @@ export class DashNavCtrl {
       });
     };
 
+    $scope.exportDashboardToPDF = function(){
+       html2canvas(document.getElementById("dashboardContainer"), {
+            onrendered: function (canvas) {
+                var data = canvas.toDataURL();
+                var docDefinition = {
+                    content: [{
+                        image: data,
+                        width: 500,
+                    }]
+                };
+                pdfMake.createPdf(docDefinition).download($scope.dashboard.title+".pdf");
```

Comment:
question - should we also add the time at which the PDF was exported? (Or the time range). @abhayakulshrestha Need your input here.